01-main/packages/marktext

```diff
@@ -0,0 +1,10 @@
+DEFVER=1
+ARCHS_SUPPORTED="amd64"
+get_github_releases "marktext/marktext" "latest"
+if [ "${ACTION}" != prettylist ]; then
+    URL="$(grep -m 1 "browser_download_url.*${TARGET_ARCH}\.deb\"" "${CACHE_FILE}" | cut -d'"' -f4)"
```

Comment:
TARGET_ARCH is not defined.  It has been used in a couple of definitions where HOST_ARCH needed to be tweaked due to inconsistent naming behaviour upstream.
```suggestion
    URL="$(grep -m 1 "browser_download_url.*${HOST_ARCH}\.deb\"" "${CACHE_FILE}" | cut -d'"' -f4)"
```